tests/test_reusable_executor.py

```diff
@@ -1037,6 +1037,45 @@ def check_faulthandler_output(
                     original_pythonfaulthandler_env
                 )
 
+    @pytest.mark.parametrize("n_cpus", [1, 2, 4])
+    def test_reusable_executor_queue_size(self, n_cpus):
+        # Check that the reusable executor can be lauched with more workers
+        # than the number of CPUs with no limitation on the queue_size.
+        #
+        # Non-regression test for https://github.com/joblib/loky/issues/396.
+
+        get_reusable_executor().shutdown()
+
+        import os
+
+        original_n_cpus = os.environ.get("LOKY_MAX_CPU_COUNT", None)
```

Comment:
Nitpick: wouldn't it be better practice to store the value of the environment variable in a global variable in the `context.py` module, and monkey patch the variable in the test (rather than changing the environment variable) ?

Reply:
This can be done in a follow up PR I guess?